Dealer.h

```diff
@@ -0,0 +1,47 @@
+#ifndef DEALER_H
+#define DEALER_H
+#include "Deck.h"
+#include <iostream>
+#include <vector>
+#include <string>
+#include <string_view>
+class Dealer
+{
+private:
```

Comment:
Somewhat a nit, somewhat not. I greatly prefer the public section to come before the private section. A lot of the time I'm just checking what the public interface of a class is, and having the private internal details upfront means I have to scroll farther to see what I'm interested in more often